packages/vue-flicking/src/Flicking.ts

```diff
@@ -149,7 +146,7 @@ export default class Flicking extends Vue {
 
   private $_getPanels(h: CreateElement) {
     const flicking = this.$_nativeFlicking;
-    const wholeSlots = this.$slots.default;
+    const wholeSlots = this.$_getSlots();
```

Comment:
@WoodNeck

It seems to be true because wholeSlots is always an array. How about using length in this case?

Reply:
Thanks for checking! I'll fix that.